domain/src/test/kotlin/cloud/fabX/fabXaccess/user/application/GettingUserTest.kt

```diff
@@ -61,7 +59,32 @@ internal class GettingUserTest {
 
         // then
         assertThat(result)
-            .isSameInstanceAs(users)
+           .isSameInstanceAs(users)
+    }
+
+    @Test
+    fun `when getting all limited then returns limited users`() = runTest {
+        // given
+        val user1 = UserFixture.arbitrary()
+        val user2 = UserFixture.arbitrary()
+        val user3 = UserFixture.arbitrary()
+
+        val users = setOf(user1, user2, user3)
+
+        val expectedLimitedUsers = setOf(
```

Comment:
I'd like to have one unit/non-integration test which really tests the "logic" of the `User.toLimitedUser()` function. I see two options:
1. Add a unit test just for the `User.toLimitedUser()`
2. Don't use the `User.toLimitedUser()` function in this test to define the `expectedLimitedUsers` but define them manually. I.e., something along the lines of 
```kotlin
val userId1 = UserIdFixture.arbitrary()
val user1 = UserFixture.arbitrary(userId1, firstName="first1", wikiName="wiki1")
[...]
val expectedLimitedUser1 = LimitedUser(userId1, "first1", "wiki1")
[...]
```